dali/operators/generic/reshape.cc

```diff
@@ -296,7 +296,8 @@ void Reshape<Backend>::CalculateOutputShape(const Workspace &ws) {
           "\n     in bytes:    ", actual_volume * input_element_size,
           "\nRequested volume: ", requested_volume,
           "\n     in bytes:    ", requested_volume * output_element_size,
-          "\nRequested shape:  ", output_shape_[i]));
+          "\nInput shape:\t", input_shape_[i],
```

Comment:
You should rebase. This change should not be visible here.

Reply:
I rebased. This line I added myself, there was no "Input shape" log there

Reply:
Oh, ok